client/src/pages/Settings/Availability/Availability.tsx

```diff
@@ -0,0 +1,311 @@
+    });
+    updateSnackBarMessage('Your new schedule was created!');
+    setNewScheduleCreated(true);
+  };
+
+  const sanitizeTimes = (schedule: any) => {
+    const sanitizedSchedule = {} as Schedule;
+    sanitizedSchedule['name'] = schedule.name;
+    daysOfTheWeek.forEach((day) => {
+      if (schedule[day].active) {
+        const startTimeInMinutes = parseInt(schedule[day].startTime) * 60;
+        const endTimeInMinutes = parseInt(schedule[day].endTime) * 60;
+
+        sanitizedSchedule[day] = {
+          active: true,
+          startTime: startTimeInMinutes,
+          endTime: endTimeInMinutes,
+        };
+      }
+    });
+    return sanitizedSchedule;
+  };
+
+  const initialDayValues = {
+    active: false,
+    startTime: '06:00',
+    endTime: '21:00',
+  };
+
+  const handleScheduleClick = (e: any, index: number) => {
+    console.log(index);
+    setSchedule(schedules[index]);
+    setShowActive(false);
+  };
+
+  //will use, need to have a function on formik onSubmit or errors out
+  const handleEditSchedule = (e: any) => {
+    console.log(e);
+  };
+
+  console.log(schedule);
+  return (
+    <div>
+      <Box
+        className={classes.container}
+        display="flex"
+        flexDirection="column"
+        justifyContent="flex-start"
+        height="100%"
+      >
+        <SettingHeader header={header} />
+        <Box className={classes.container} display="flex" flexDirection="row" justifyContent="flex-start">
+          <FormControl className={classes.select}>
+            <InputLabel id="scheduleName">Schedules</InputLabel>
+            <Select id="scheduleName" name="schedule">
+              {schedules &&
+                schedules.map((schedule, index) => {
+                  return (
+                    <MenuItem
+                      onClick={(e) => {
+                        handleScheduleClick(e, index);
+                      }}
+                      key={schedule._id}
+                      id={schedule._id}
+                      value={schedule._id}
+                    >
+                      {schedule.name}
+                    </MenuItem>
+                  );
+                })}
+            </Select>
+          </FormControl>
+          <Button
+            onClick={() => handleNewSchedule()}
+            sx={{ borderRadius: '10%', color: 'green', border: 'solid 2px green' }}
+            className={classes.addSchedule}
+          >
+            Add Schedule +
+          </Button>
+        </Box>
+        {schedule && (
+          <ScheduleForm
+            schedule={schedule}
+            handleEditSchedule={handleEditSchedule}
+            showActive={showActive}
+            handleMakeActiveSchedule={handleMakeActiveSchedule}
+          ></ScheduleForm>
+        )}
+        {newSchedule && !schedule && (
+          <>
+            <Formik
+              initialValues={{
+                name: '',
+                monday: {
+                  active: initialDayValues.active,
```

Comment:
We could store these fields in an object as initialDay and reuse it rather than having to repeat ourselves